### PR TITLE
Adding title to readme.md file

### PR DESCRIPTION
**Pull request by kshithijiyer**

### CODE REVIEW

**Review by dustymabe — APPROVED** [review]

**Comment by dustymabe** [comment]
Thanks @kshithijiyer

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+# Fedora Coreos Config
```
Comment (arithx) [inline README.md:1]: It should be `Fedora CoreOS Config`

Comment (dustymabe) [reply README.md:1]: yeah @kshithijiyer want to fix it?

Comment (kshithijiyer) [reply README.md:1]: Done #140

**Pull request merged**